packages/create-sitecore-jss/src/templates/react/package.json

```
@@ -0,0 +1,122 @@
+{
+  "name": "jss-sample-react",
```

```suggestion
  "name": "<%- appName %>",
```
this file needs token replacement for prefix and appName